### DISCUSSION TITLE
Is this still active?

### DISCUSSION THREAD

**Discussion by digital-dick** [cea-sec/ivre#1555] [General]
Just asking if this is still active really? If so, i have some questions :)

1) Ive downloaded it and hosting it on a webserver publicly, but noticed it has some CVE's. Does IVRE get updated?

2) Great tool by the way, in a week ive managed to scan over a 100,000, and noticed that the ivre scan2db take a while now, just wondered what the db limit is, can it go to a million or am I being hopeful? 

Keep up the good work!

**Comment by p-l-** [comment]
1. Last commit 1 week ago, I'd say the project is active yes! What CVEs does it have?
2. IVRE scales as well as its database. If you use MongoDB, you might have to upgrade the hardware it runs on or use sharding. Please refer to MongoDB's documentation on how to scale.
Hope this helps!
